core/src/main/java/org/eclipse/hono/config/ClientConfigProperties.java

```diff
@@ -555,6 +557,30 @@ public final void setTlsEnabled(final boolean enabled) {
         this.tlsEnabled = enabled;
     }
 
+    /**
+     * Get the host override for TLS SNI Server Name set by {@link #setSniServerName(String)}.
+     *
+     * @return  the sniServerName
+     */
+    public final String getSniServerName() {
+        return sniServerName;
+    }
+
+    /**
+     * Explicitly override the host to use for the TLS SNI server name.
+     *
+     * If neither the {@link ClientConfigProperties#setHost(String) host} or SNI server name is explicitly
+     * overridden, the host specified in {@link ClientConfigProperties#getHost()} will be used, with SNI performed implicitly
+     * where a FQDN was specified.
+     *
+     * This method should typically only be needed to set different values for the host and SNI server name.
+     *
+     * @param sniServerName to set as SNI server name
```

Comment:
Please start the description with an upper case letter and finish with a period, e.g. 
> `@param` sniServerName The host name to use in the SNI extension.